app/View/Pages/home.ctp

```diff
@@ -58,19 +58,19 @@ endif;
 	$settings = Cache::settings();
 	if (!empty($settings)):
 		echo '<span class="notice success">';
-			echo __d('cake_dev', 'The %s is being used for core caching. To change the config edit %s', '<em>' . $settings['engine'] . 'Engine</em>', 'APP/Config/core.php');
+			echo __d('cake_dev', 'The %s is being used for core caching. To change the config edit %s', '<em>' . $settings['engine'] . 'Engine</em>', 'CONFIG/core.php');
```

Comment:
Is this deliberate?
It would be more helpful if it was like ``CONFIG . 'core.php'``